package.json

```diff
@@ -187,7 +187,7 @@
         "yarn i18n",
         "git add locales"
       ],
-      "after:bump": "pipx run towncrier build --draft --yes --version ${version} > .changelog.draft && pipx run towncrier build --yes --version ${version} && make corepackagebump VERSION=${version}",
+      "after:bump": "pipx run towncrier build --draft --yes --version ${version} > .changelog.draft && pipx run towncrier build --yes --version ${version} && make corepackagebump VERSION=${version} && cp CHANGELOG.md docs/source/release-notes/index.md && git add docs/source/release-notes/index.md && git commit -m 'copy CHANGELOG.md' && git push",
```

Comment:
I think you can leave out the git add, git commit, and git push here. release-it already pushes a commit with the changes that are made during this hook.

Reply:
I'd move it to a makefile command (as the `corepackagebump`). In fact, the `corepackagebump` actions are similar to these ones.